### PR TITLE
Makes Ratvar a they (plural)

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: hmm
fix: hmm
/:cl:

### CODE REVIEW

**Comment by vcordie** [comment]
hmm

**Comment by Anonmare** [comment]
Hmm

**Comment by RemieRichards** [comment]
Goof, don't make any more PRs with titles like this, I'm just going to close them from now on.

**Comment by GunHog** [comment]
Stupid meme name and changelog aside (why are we tolerating this?), Ratvar is often referred as male, so use MALE instead of PLURAL.

**Comment by ChangelingRain** [comment]
You get one chance to change this to `gender = MALE`

**Comment by Iamgoofball** [comment]
@ChangelingRain Ratvar can identify as what they want. Who are you to define their gender for them?

**Comment by Iamgoofball** [comment]
okay there you go

File: code/game/gamemodes/clock_cult/clock_structures/ratvar_the_clockwork_justicar.dm [hunk 1]
```diff
@@ -29,6 +30,9 @@
 	var/area/A = get_area(src)
 	notify_ghosts("The Justiciar's light calls to you! Reach out to Ratvar in [A.name] to be granted a shell to spread his glory!", null, source = src, alert_overlay = alert_overlay)
 	INVOKE_ASYNC(SSshuttle.emergency, /obj/docking_port/mobile/emergency..proc/request, null, 0, 0)
+	var/chance = 1 // one chance
+	if(prob(chance))
+		gender = MALE
```
Comment (Jalleo) [inline code/game/gamemodes/clock_cult/clock_structures/ratvar_the_clockwork_justicar.dm]: Heh.

**Comment by ChangelingRain** [comment]
You have failed your one chance.

**Comment by Iamgoofball** [comment]
@optimumtact @KorPhaeron help my lore's being violated

**Comment by Iamgoofball** [comment]
also @Cheridan

**Pull request closed without merging**